lib/BloqadeSchema/src/validate.jl

```diff
@@ -0,0 +1,203 @@
+
```

Comment:
validate shouldn't throw error but throw an error in `to_schema'. create struct that contains all error messages and overload `Base.show` to print the messages out in a nice way. 

https://docs.julialang.org/en/v1/manual/types/#man-custom-pretty-printing